core/vm/contracts.go

```diff
@@ -103,6 +103,7 @@ var PrecompiledContractsApricotPhase2 = map[common.Address]StatefulPrecompiledCo
 	genesisContractAddr:              &deprecatedContract{},
 	nativeAssetBalanceAddr:           &nativeAssetBalance{gasCost: params.AssetBalanceApricot},
 	nativeAssetCallAddr:              &nativeAssetCall{gasCost: params.AssetCallApricot},
+	exportAssetAddr:                  &exportAsset{gasCost: params.AssetCallApricot},
```

Comment:
I think it should have it's own gas cost, equal to the cost of an export transaction, no?

Reply:
You are most likely correct. I haven't yet figured out what the gas cost here should be, as the `newExportTx` method also dynamically allocates gas.

I am starting to think there shouldn't be a hardcoded gas cost here. Either it's zero, because it's handled elsewhere, or it should use/return the gas determined by the `newExportTx` method. Thoughts?

Reply:
It should not be zero. I believe precompiles must have a fixed gas cost. Typical export cost is 0.001 AVAX. With 25 nAVAX / gas as the typical price on the C chain, gas cost should be somewhere in the range of 10000-40000.